cinema/views.py

```diff
@@ -1 +1,63 @@
-# write views here
+from rest_framework import viewsets, serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+from cinema.serializers import (GenreSerializer,
+                                ActorSerializer,
+                                CinemaHallSerializer,
+                                MovieSerializer,
+                                MovieSessionSerializer,
+                                MovieListSerializer,
+                                MovieDetailSerializer,
+                                MovieSessionListSerializer,
+                                MovieSessionDetailSerializer)
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_queryset(self) -> queryset:
+        queryset = self.queryset
```

Comment:
seems like this variable is redundant(and it doesn't simplify your code). You can just use `self.queryset`.
e.g.
```python
if self.action in ...:
    return self.queryset....
return self.queryset
```